bigquery/google/cloud/bigquery/job.py

```diff
@@ -506,6 +529,11 @@ def output_rows(self):
     https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.allowQuotedNewlines
     """
 
+    autodetect_schema = AutoDetectSchema('autodetect_schema', bool)
```

Comment:
You can add a second commit and push to your branch with fixes. GitHub allows for "Squash and merge" now, so it's no big deal if the git history gets a little messy in the PR.